state/redis/metadata.yaml

```
@@ -40,11 +40,13 @@ metadata:
     required: false
     description: Maximum number of retries before giving up. Defaults to \"3\".
     example: "5"
+    default: "3"
     type: number
   - name: maxRetryBackoff
     required: false
-    description: Maximum backoff between each retry. Defaults to 2 seconds; \"-1\" disables backoff.
+    description: Maximum backoff between each retry in milliseconds. Defaults to 2 seconds; \"-1\" disables backoff.
```

Is it milliseconds or nanoseconds? Given the number of 0's, looks nano

---

You are right, all time.Duration is in nanoseconds. Let me address this.

```suggestion
    description: Maximum backoff between each retry in nanoseconds. Defaults to 2 seconds; \"-1\" disables backoff.
```